<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePromotion</class>
 <widget class="QWidget" name="CreatePromotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Promotion</string>
  </property>
  <widget class="QLabel" name="label_Heading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Create A New Promotion</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="50,100,70">
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Expiry">
      <property name="text">
       <string>Expiry Date(optional)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Expiry"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="error_Value">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Barcode">
      <property name="text">
       <string>Barcode</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Barcode"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="error_Barcode">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="error_Expiry">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Value"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Value">
      <property name="text">
       <string>Promotion Bundle Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QPushButton" name="create">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>CreatePromotion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
